src/components/footer.js

```diff
@@ -13,25 +14,18 @@ import logoImg from "../images/logos/cto_logo_horizontal_white.png";
 function Footer() {
   const { t } = useTranslation();
   const primaryLinks = ["/link-one", "/link-two", "/link-three", "/link-four"];
```

Comment:
I think the links should be in the translation.json or a file in the constants folder. Probably the translation.json since other information about the links is there

Reply:
Keeping it `translation.json`would be duplication because the user will be directed to the same route no matter what language they're reading in. But file in the constants folder, good idea!